### ISSUE TITLE
Error Applying Baked Templates

### ISSUE THREAD

**Issue by Einwig** [FlailingFog/KK-Blender-Porter-Pack#109]
Been having an issue when I get to the Apply Baked Templates step. I tried with shaders edited and unedited, and both yield the same result.
![blender_HYhe755xOq](https://user-images.githubusercontent.com/99285963/153056082-b2239112-1605-49bf-b363-d93d1b021659.png)

It looks as if it's applying a body alphamask, but the clothes and hair just straight up disappear.

Here is what the model is supposed to look like (without any edited shaders in this test):

![blender_Od3ZkEASIK](https://user-images.githubusercontent.com/99285963/153056264-319a0411-6441-401f-8086-fca480a5e7ca.png)

The baked materials look like they should be fine. The pants are top row, shirt is 3rd row, and hairs are throughout. None of them look particularly problematic, though I'm not entirely familiar with Blender enough to tell.

**Comment by FlailingFog** [comment]
If the baked materials in the folder look correct, then they must not be loading in correctly. You're not getting any errors, right? And are there any images loaded into the Image Texture node that gets added to the bottom of each material template or is it blank? It should be filled in for each material
![image](https://user-images.githubusercontent.com/65811931/153103662-2549e614-11f4-4df9-b828-709dc7495675.png)

**Comment by Einwig** [comment]
Assuming these are what you're referring to:
![blender_V5NY1xmAgF](https://user-images.githubusercontent.com/99285963/153106225-45176cac-32ad-4170-8a8a-987934a21a87.png)
![blender_lS7w0tnduL](https://user-images.githubusercontent.com/99285963/153106230-a0b7a2a0-d2af-4495-ad30-f652e6526bb6.png)

All the hairs, the pants, and the shirt indeed have the images textures slot filled

**Comment by Einwig** [comment]
Wait no I misunderstood, I thought you meant before applying. After applying the baked templates, the textures were not automatically set.

Here is with the shirt and pants manually applied, but the hairs are gonna be a pain (8+ pieces)
![blender_OKMzr63D2B](https://user-images.githubusercontent.com/99285963/153107887-bf1525c9-2c30-47d4-a7cb-2c31f99bbe9b.png) 


Is there an automatic way to remedy this, or do I have to do this manually?

**Comment by FlailingFog** [comment]
It should be done automatically, not sure why it isn't loading them in. You're going *inside* of the folder you stored the baked materials into before you confirm, right?

**Comment by Einwig** [comment]
Yes. On the prompt, I open the folder that contains it, then press the [open baked materials folder] button

**Comment by Einwig** [comment]
As a side note, in the middle of applying the textures manually, the second eye color was not baked at all.

**Comment by Einwig** [comment]
sigh, the wonders of talking before you tried everything. The correct second color did bake, but just wasn't showing up in the drop-down menu of the box. Sorry about that.

**Comment by FlailingFog** [comment]
So the baked textures are loading into Blender but not automatically being applied... Could I have the card to try it?

**Comment by Einwig** [comment]
Sure, but I would prefer to have as few people with this card as possible. Could I send it privately?

**Comment by MediaMoots** [comment]
Did you try using a different card to see if this issue is/isn't specific to your card?

**Comment by FrankVargas427** [comment]
Since this issue is still open, I'd like to comment on my case. Literally, not a single texture is baked. All of them fail. And as I combed through the textures to manually add them to the new node that appeared, this is what I discovered.

![image](https://user-images.githubusercontent.com/43286109/159398651-42e76c10-40a5-4172-a4b4-681e317de696.png)

Something about the naming scheme fails and for some reason it puts  <UDIM> instead of the number 1024. Doing a bit of a reload on the image or selecting it again solves the issue, but it still is weird that instead of reading 1024 on the file, it reads UDIM. Renaming the file to have it not have any numbers or so someone else altogether makes it work, but again, none of the files are automatically found and applied.
